content/authors/Yiwei/index.md

```diff
@@ -0,0 +1,36 @@
+---
+# Display name
+title: Yiwei Huang
+
+# Username (this should match the folder name)
+authors:
+- yiwei14
+
+# Is this the primary user of the site?
+superuser: false
+
+# Role/position
+role: Graduate Student
+
+Organizations/Affiliations:
+organizations:
+- name: University of Alberta
+  
+
+# Short bio (displayed in user profile at end of posts)
+bio: 
+
+interests:
+- power analysis
+
+
+education:
+  courses:
+  - course:Statistics and Economics
```

Comment:
I think there should  be a space after `:`

Reply:
Thanks so much! 
It works now!